src/warehouse/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "dotnet.defaultSolution": "WarehouseService.sln"
```

Comment:
This should be ignored: https://github.com/github/gitignore/blob/main/Global/VisualStudioCode.gitignore

Reply:
Fixed